templates/producer.properties.j2

```diff
@@ -25,7 +25,8 @@ bootstrap.servers={{ kafka_producer_bootstrap_servers }}
 # specify the compression codec for all data generated: none, gzip, snappy, lz4, zstd
 compression.type={{ kafka_producer_compression_type }}
 
-# name of the partitioner class for partitioning events; default partition spreads data randomly
+# name of the partitioner class for partitioning records;
```

Comment:
Updates to comments based on diff'ing the latest config from the Kafka 3.2.0 release.